tests/platform_tests/test_reboot.py

```diff
@@ -192,6 +192,10 @@ def test_soft_reboot(duthosts, enum_rand_one_per_hwsku_hostname,
     if duthost.is_multi_asic:
         pytest.skip("Multi-ASIC devices not supporting soft reboot")
 
+    if "s6100" not in duthost.facts['platform']:
```

Comment:
There is better (faster and standard) way of skipping a test based on a condition. Check below file where skip conditions are defined.

That way will prevent the test from even starting and getting to this point.

Reply:
Modified accordingly. Thank you!